beep/structure.py

```diff
@@ -2037,7 +2037,12 @@ def determine_paused(group, paused_threshold=3600):
         for t in date_time_obj
     ]
     date_time_float = pd.Series(date_time_float)
-    return int(date_time_float.diff().max() > paused_threshold)
+    if date_time_float.diff().max() > paused_threshold:
+        paused_secs = date_time_float.diff().max()
+    else:
+        paused_secs = 0
+
```

Comment:
I think I can see Will's source of confusion here. I suppose `paused_secs` could be renamed `max_pause_duration`, since you are only returning the `max` if there is >1 pause exceeding the threshold duration.

Reply:
Ah, that would definitely explain it, thanks Chirru!